assets/js/common/ApiKeySettingsModal/ApiKeySettingsModal.jsx

```diff
@@ -39,6 +40,8 @@ const availableTimeOptions = [
 ];
 
 function ApiKeySettingsModal({
+  userAbilities,
+  permitted,
```

Comment:
You don't need to protect the modal, you protected to button to open the modal and backend is already protected, no guards needed here